pipeline/README.md

```diff
@@ -0,0 +1,90 @@
+# Pipeline
+
+This folder should contain all code and resources required for the etl pipeline, which Connects to the api, extracts all relevant plant data 
+cleans and processes the data for upload and them uploads the data to the rds. 
+
+
+### Files
+
+- `extract.py`
+  - python script that loops through the api plant pages extracting all the relevant data and then collates the data into a pandas DataFrame ready for cleaning and other activities. 
+
+- `transform.py`
+  - This script analyses (sends alert emails), cleans and processes the passed DataFrame for upload.
+
+- `load.py`
+  - This script uploads the recordings data to the rds. 
+
+- `etl.py`
+  - python script that connects and runs the previous files.
+
+- `emailing.py`
+  - A python script that builds the email structure and content using html, querying the rds for relevant details and then sending the email to relevant botanists.
+
+- `lambda_function.py`
+  - Formats the pipeline for use with AWS lambda.
+
+- `requirements.txt`
+  - A text file containing all the dependencies needed for the folder to run.
+
+- `Dockerfile`
+  - Docker file to build an image of the pipeline.
+
+
+### Development 
+
+You will nee a .env file with the following structure.
+
+```
+AWS_ACCESS_KEY=x
+AWS_SECRET_ACCESS_KEY=x
+DB_HOST=x
+DB_PORT=x
+DB_NAME=x
+DB_USER=x
+DB_PASSWORD=x
+SCHEMA_NAME=x
+FROM_EMAIL=x
+TO_EMAIL1=x
+TO_EMAIL2=x
+TO_EMAIL3=x
+TO_EMAIL4=x
```

Comment:
```suggestion
AWS_ACCESS_KEY=XXXXXXXXXXXXXXX
AWS_SECRET_ACCESS_KEY=XXXXXXXXXXXXXXX
DB_HOST=XXXXXXXXXXXXXXX
DB_PORT=XXXXXXXXXXXXXXX
DB_NAME=XXXXXXXXXXXXXXX
DB_USER=XXXXXXXXXXXXXXX
DB_PASSWORD=XXXXXXXXXXXXXXX
SCHEMA_NAME=XXXXXXXXXXXXXXX
FROM_EMAIL=XXXXXXXXXXXXXXX
TO_EMAIL1=XXXXXXXXXXXXXXX
TO_EMAIL2=XXXXXXXXXXXXXXX
TO_EMAIL3=XXXXXXXXXXXXXXX
TO_EMAIL4=XXXXXXXXXXXXXXX
```

It just looks nicer. Also why do you only allow 4 emails to receive notifications? Could you not have done:
```
TO_EMAILS=[EMAIL1,EMAIL2,EMAIL3,EMAIL4] 
```
and then make it a list?